include/dem/dem_solver_parameters.h

```diff
@@ -70,6 +72,12 @@ class DEMSolverParameters
     simulation_control.parse_parameters(prm);
     floating_walls.parse_parameters(prm);
     boundary_motion.parse_parameters(prm);
+    forces_torques.parse_parameters(prm);
+    if (boundary_motion.motion_method==Parameters::Lagrangian::BoundaryMotion<dim>::MotionMethod::free)
+    {
+      physical_properties.g[1]=physical_properties.g[1]*cos(forces_torques.angle_theta*M_PI/180)-physical_properties.g[2]*sin(forces_torques.angle_theta*M_PI/180);
+      physical_properties.g[2]=physical_properties.g[2]*cos(forces_torques.angle_theta*M_PI/180)+physical_properties.g[1]*sin(forces_torques.angle_theta*M_PI/180);
+    }
```

Comment:
This is a bit messy no? I would leave gravity alone, do you need the angle for anything actually if you set the gravity in Diagonal? It'S better to let the user manually fix the gravity at an angle than the other way around.

Reply:
I wanted the user to just have to choose about the slope's angle. And put the actual gravity in cartesian axis.